app/assets/javascripts/views/story_view.js

```diff
@@ -403,39 +404,45 @@ Fulcrum.StoryView = Fulcrum.FormView.extend({
               window.md.makeHtml(this.model.escape('description'))
             );
             $(div).append(description);
-            $(description).after(
-              this.make('input', {
-                class: 'edit-description',
-                type: 'button',
-                value: I18n.t('edit')
-              })
-            );
+            if (!this.model.get('description') || 0 === this.model.get('description').length) {
```

Comment:
AUSHUHUAS, it's a good way to avoid forgetting a `=` and end up doing something like `this.model.get('description').length = 0` though.